utils/hwloc/hwloc-calc.1in

```diff
@@ -534,6 +536,23 @@ OS devices may also be filtered by subtype. In this example, there are
     $ utils/hwloc/hwloc-calc -I coproc node:1
     7,8
 
+Example of use of the systemd-dbus-api cpuset output format:
```

Comment:
Start a new man section, the EXAMPLES one is already too long. Something like EXAMPLES - SYSTEMD DBUS API. I split the rest of examples later.

Reply:
I pushed some changes.